### PR TITLE
Bugfixes Bayesian correlation

### PR DESCRIPTION
**Pull request by TimKDJ**
Bayesian correlation matrix:
- logBF used base 10 instead of e (#1981)
- flagging of supported correlations was incorrect for BF01 

Bayesian correlation pairs:
- footnotes vanished when calculations were retrieved from state
- plots showed the incorrect BF when LogBF10 was selected

### CODE REVIEW

File: JASP-Engine/JASP/R/correlationbayesian.R [hunk 1]
```diff
@@ -239,9 +239,11 @@ CorrelationBayesian <- function(dataset=NULL, options, perform="run",
 	
 	if (flagSupported) {
 		if (bayesFactorType=="LogBF10") {
-			.addFootnote(footnotes, paste(bfTitle, " > log(10), ** , ", bfTitle, " > log(30), *** ", bfTitle, " > log(100)"), symbol="*")
+			.addFootnote(footnotes, paste(bfTitle, " > log(10), ** ", bfTitle, " > log(30), *** ", bfTitle, " > log(100)"), symbol="*")
+		} else if (bayesFactorType == "BF01") {
+			.addFootnote(footnotes, paste(bfTitle, " < .1, ** ", bfTitle, " < .03, *** ", bfTitle, " < .01"), symbol="*")
```
Comment (AlexanderLyNL) [inline JASP-Engine/JASP/R/correlationbayesian.R]: .addFootnote(footnotes, paste(bfTitle, " < 10, ** ", bfTitle, " < 30, *** ", bfTitle, " < 100"), symbol="*")

File: JASP-Engine/JASP/R/correlationbayesianpairs.R [hunk 2]
```diff
@@ -762,7 +768,7 @@ CorrelationBayesianPairs <- function(dataset=NULL, options, perform="run", callb
 							# 
 							# plot[["data"]] <- .endSaveImage(image)
 							.plotFunc <- function() {
-								.plotPosterior.correlation(r=rs[i], n=ns[i], oneSided=oneSided, BF=BF10post[i], BFH1H0=BFH1H0, addInformation=options$plotPriorAndPosteriorAdditionalInfo, kappa=options$priorWidth,corCoefficient=options$corcoefficient)
+								.plotPosterior.correlation(r=rs[i], n=ns[i], oneSided=oneSided, BF=plotBF10post, BFH1H0=BFH1H0, addInformation=options$plotPriorAndPosteriorAdditionalInfo, kappa=options$priorWidth,corCoefficient=options$corcoefficient)
```
Comment (AlexanderLyNL) [inline JASP-Engine/JASP/R/correlationbayesianpairs.R]: What's the difference between BF10post[i] and plotBF10post?

File: JASP-Engine/JASP/R/correlationbayesian.R (hunk 1 above)
Comment (TimKDJ) [reply JASP-Engine/JASP/R/correlationbayesian.R]: We're looking for evidence for model "1" though.
Let's say:
marginal model 0: 100
marginal model 1: 500

BF01 = 0.2
BF10 = 5

As marginal of 1 increases we gain more evidence and so if BF01 < 0.1 add a star, or if BF10 > 10 add a star. Right?

File: JASP-Engine/JASP/R/correlationbayesianpairs.R (hunk 2 above)
Comment (TimKDJ) [reply JASP-Engine/JASP/R/correlationbayesianpairs.R]: It is the exponent of logBF or otherwise just BF

**Pull request merged**